contentcuration/contentcuration/serializers.py

```diff
@@ -68,17 +69,23 @@ def get(*args, **kwargs):
     def get_file_url(self, obj):
         return obj.file_on_disk.url
 
+    def get_storage_path(self, obj):
```

Comment:
I tried using that, but got an assertion error: `It is redundant to specify 'get_storage_url' on SerializerMethodField 'storage_url' in serializer 'FileSerializer', because it is the same as the default method name. Remove the `method_name` argument`